### PR TITLE
Add support for starky

### PR DESCRIPTION
**Pull request by tumberger**
Provides support for starky for:
- SHA256
- Exponentiate
- Memory & Time benchmarks

Issues to still resolve:
- [X] commands_merge overwrites all memory benchmarks with last invoked. EDIT - See #32 
- [X] memory for verification is currently not measured due to missing serialization / deserialization
- [X] proof size is not added due to missing serialization into tmp folder when invoked with "prove"

### CODE REVIEW

**Comment by tumberger** [comment]
TODO:
- [X] Fix inputs Exponentiate2 -> 2,4,8,16,32...2^23, parse as power of 2 directly
- [X] Setup - currently not benchmarked. Figure out where it actually is to benchmark / include / exclude (part of prove?)
 - NOTE - Setup not benchmarked due to starky public randomness chosen by verifier.

File: _scripts/reader/process_circuit.py [hunk 1]
```diff
@@ -208,6 +208,108 @@ def build_command_bellman(payload, count):
...
+    # TODO - Add count to command creation
+    if len(payload.backend) != 1 or payload.backend[0] != "starky":
+        raise ValueError("Starky benchmark only supports starky backend")
+    # TODO - Solution for Starks - don't use curve, rename parameter / other option?
+    if len(payload.curves) != 1 or payload.curves[0] != "goldilocks":
+        raise ValueError("Starky benchmark only supports goldilocks field")
+    # TODO - Support verification memory benchmarks - problem deserialization
+    for op in payload.operation:
+        if op == "verify":
+            user_input = input("Verify Memory Benchmarks currently not supported, if you'd still like to continue press any key, else press 'n' to stop:")
```
Comment (StefanosChaliasos) [inline _scripts/reader/process_circuit.py]: I suggest removing user input and just leave a TODO

File: _input/circuit/exponentiate_2/input_10.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "X": "1",
+    "E": "10",
```
Comment (StefanosChaliasos) [inline _input/circuit/exponentiate_2/input_10.json]: I suggest directly use `2^10` i.e., 1024 here and update the code accordingly

**Comment by tumberger** [comment]
Fixed the above two issues @StefanosChaliasos

**Pull request merged**